File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/TransferDeliveryPackageRequiredlabtestbatch.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class TransferDeliveryPackageRequiredlabtestbatch
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestBatchId")]
        public required int LabTestBatchId { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("LabTestBatchName")]
        public required string LabTestBatchName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PackageId")]
        public required int PackageId { get; set; }
    }
}

File: sdks/thunkmetrc-go/wrapper/models/transfer_delivery_package_requiredlabtestbatch.go
package models

type TransferDeliveryPackageRequiredlabtestbatch struct {
    LabTestBatchId int64 `json:"LabTestBatchId,omitempty"`
    LabTestBatchName string `json:"LabTestBatchName,omitempty"`
    PackageId int64 `json:"PackageId,omitempty"`
}
